### ISSUE TITLE
ingestors use a mixture of Python 2 and Python 3; should use Python 3 for all modules

### ISSUE THREAD

**Issue by bozyurt** [biocaddie/data-pipeline#1]
ingestors use a mixture of Python 2 and Python 3. It should use Python 3 for all modules.
Also do not use spaces in file and directory names for Unix/Linux.

**Comment by jgrethe** [comment]
Hi Ted and Muhammad - do you have a Unix/Linux system to test these on?

**Comment by MAK016** [comment]
Yeah I have a Linux system I develop on. Ill clean up the file names and change all code to be python3 compatible.

**Comment by Teddochi** [comment]
I am working on a windows system.
